braindecode/datasets/sleep_physionet.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import numpy as np
+import pandas as pd
+import mne
+from mne.datasets.sleep_physionet.age import fetch_data
+
+from .base import BaseDataset, BaseConcatDataset
+
+
+class SleepPhysionet(BaseConcatDataset):
+    """Sleep Physionet dataset.
+
+    Sleep dataset from https://physionet.org/content/sleep-edfx/1.0.0/.
+    Contains overnight recordings from 78 healthy subjects.
+
+    See [MNE example](https://mne.tools/stable/auto_tutorials/sample-datasets/plot_sleep.html).
+
+    Parameters
+    ----------
+    subject_ids: list(int) | int
+        (list of) int of subject(s) to be loaded.
+    recording_ids: list(int)
+        Recordings to load per subject (each subject except 13 has two
+        recordings). Can be [1], [2] or [1, 2].
+    preload: bool
+        if True, preload the data of the Raw objects.
+    load_eeg_only: bool
+        If True, only load the EEG channels and discard the others (EOG, EMG,
+        temperature, respiration) to avoid resampling the other signals.
+    crop_wake_mins: float
+        Number of minutes of wake time to keep before the first sleep event
+        and after the last sleep event. Used to reduce the imbalance in this
+        dataset. Ignore if
```

Comment:
if?

Reply:
I think you should default to the common thing ie 5 mins before and 5 mins after sleeping.